src/main/java/reposense/commits/CommitInfoExtractor.java

```diff
@@ -59,9 +60,9 @@
             Matcher matcher = TRAILING_NEWLINES_PATTERN.matcher(rawCommitInfos[i]);
             String rawCommitInfo = matcher.replaceAll("");
 
-            int statLineSeparatorIndex = rawCommitInfo.lastIndexOf("|");
+            int statLineSeparatorIndex = rawCommitInfo.lastIndexOf(INFO_STAT_SEPARATOR);
             String infoLine = rawCommitInfo.substring(0, statLineSeparatorIndex);
-            String statLine = rawCommitInfo.substring(statLineSeparatorIndex + 1);
+            String statLine = rawCommitInfo.substring(statLineSeparatorIndex + 1).trim();
```

Comment:
`.trim()` is to remove the first new line character in `rawCommitInfo.substring(statLineSeparatorIndex + 1)`